lxd/container.go

```diff
@@ -738,6 +720,7 @@ func containerCreateInternal(d *Daemon, args containerArgs) (container, error) {
 	args.CreationDate = dbArgs.CreationDate
 	args.LastUsedDate = dbArgs.LastUsedDate
 
+	// Setup the container struct and finish creation (storage and idmap)
```

Comment:
What do you mean?

The pool is validated and the volume added to the database in containerLXCCreate so the comment seems accurate to me.